src/main/java/com/aspose/imaging/cloud/sdk/model/requests/CreateImageFrameRangeRequest.java

```diff
@@ -0,0 +1,134 @@
+/*
+* --------------------------------------------------------------------------------------------------------------------
+* <copyright company="Aspose" file="CreateImageFrameRangeRequest.java">
+*   Copyright (c) 2018-2019 Aspose Pty Ltd. All rights reserved.
```

Comment:
Please, change the copyright period to 2020 in all the new classes

Reply:
I've updated the copyright dates.